### PR TITLE
Update n-body

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

**Comment by dcodeIO** [comment]
For comparison:

```
COLD SERIES:

Performing 20000000 steps (AssemblyScript WASM) ...
Took 1515.6276ms
Performing 20000000 steps (AssemblyScript JS) ...
Took 2409.551ms
Performing 20000000 steps (JS) ...
Took 1826.1667ms
Performing 20000000 steps (Rust WASM) ...
Took 1542.6844ms

WARMED UP SERIES:

Performing 20000000 steps (AssemblyScript WASM) ...
Took 1518.5098ms
Performing 20000000 steps (AssemblyScript JS) ...
Took 2383.6588ms
Performing 20000000 steps (JS) ...
Took 1806.5908ms
Performing 20000000 steps (Rust WASM) ...
Took 1542.4533999999999ms
```

**Comment by MaxGraey** [comment]
could we merge this?

**Comment by dcodeIO** [comment]
Do you have a suspicion why it is slower now? Given that it still uses the stub runtime, that rather looks like a regression unrelated to the latest changes.

**Comment by MaxGraey** [comment]
wasm module doesn't changed at all. So perhaps it's slightly slower due to some v8 regressons

File: n-body/assembly/index.ts [hunk 1]
```diff
@@ -193,13 +190,13 @@ export function init(): void {
...
-export function step(): float {
+export function step(): f64 {
   system.advance(0.01);
   return system.energy();
 }
 
-export function bench(steps: u32): void {
-  for (let i: u32 = 0; i < steps; ++i) system.advance(0.01);
+export function bench(steps: i32): void {
+  for (let i = 0; i < steps; ++i) system.advance(0.01);
```
Comment (dcodeIO) [inline n-body/assembly/index.ts:199]: Now that you mention that the binary didn't change, shouldn't this change at least have some effect?

Comment (dcodeIO) [reply n-body/assembly/index.ts:199]: Ah, no, implicitly converts to the LHS type anyway, I see.

**Pull request closed without merging**